### ISSUE TITLE
Nothing to undo

### ISSUE THREAD

**Issue by x1p** [SBPrime/AsyncWorldEdit-Premium#245]
I've setup AWE so that it remembers undo actions to a certain time.
After reconnecting, AWE refuses to undo. `Nothing to undo.`

The /undo/my-uuid-folder contains my files alright. It makes 4 files per action.
As long as I don't rejoin, the undo action is performed.
But my goal is to accept undo actions for a certain amount of time, regardless of rejoining.
This due to staff must be able to switch servers if priority asks for it.
When they return to the original server they should be able to pickup where they left.

[config.txt](https://github.com/SBPrime/AsyncWorldEdit-Premium/files/5740145/config.txt)

AWE 3.8.1
WE 7.2.0
Paper 1.16.3-1.16.4

**Comment by x1p** [comment]
FYI: if AWE is loaded, it always says `0 blocks affected`
Purely WE does count affected block correctly

**Comment by SBPrime** [comment]
Please clarify one thing: normal undo works?
This option should allow you to do what you need:
```
#How much time should the WorldEdit session be kept after player logged out
#Values: -1 Do not remove, 0 - immediately, X - Number of minutes
keepSessionOnLogoutFor: 60
````

For this option to work properly the user UUID needs to stay the same between logout/login.

**Comment by x1p** [comment]
Normal undo works, as long as the player is online and stays online.

I attached my config, so you could see my settings.
I have had for a really long time this value: `keepSessionOnLogoutFor: 60` which used to work fine.
I needed it a few days ago and I noticed it didn't work. I had to recover a backup in order to undo this action.

The UUID is always the same. Server is in online mode through bungee

**Comment by SBPrime** [comment]
Nothing regarding the undo changed, could you do som tests on your end? a small edit + logout + login + undo would be enough. What operation did you try to undo? you can also try setting awe to debug mode and checking the logs for undo storage messages. I would also suggest doing this:

1. clean the undo dir for player
2. login
2. small edit
3. check the undo dir for player
2. logout
3. login
4. 2nd small edit
5. check the undo dir for player
5. undo
6. undo
7. check the undo dir for player

**Comment by x1p** [comment]
> Nothing regarding the undo changed, could you do some tests on your end?

The bug probably exists for a while. I didn't try older versions.

Regarding the test:
1:
![Screenshot_1](https://user-images.githubusercontent.com/28690790/103410164-71a22380-4b6a-11eb-88f5-f39cacc2a7ad.png)

4:
![Screenshot_4](https://user-images.githubusercontent.com/28690790/103410174-79fa5e80-4b6a-11eb-9ac5-72aa6367c261.png)

8:
![Screenshot_8](https://user-images.githubusercontent.com/28690790/103410182-7ff03f80-4b6a-11eb-8ae1-d6d2d5de1aa4.png)

9:
![Screenshot_9](https://user-images.githubusercontent.com/28690790/103410300-09077680-4b6b-11eb-82b1-da416aaae3ac.png)

10:
To my surprise, this time he said he had undone something, but failed to remove 
![Screenshot_10](https://user-images.githubusercontent.com/28690790/103410311-16bcfc00-4b6b-11eb-8e20-97ddb2e561b4.png)


11:
![Screenshot_11](https://user-images.githubusercontent.com/28690790/103410193-87174d80-4b6a-11eb-8e38-734d39019cd9.png)

12:
Tried to undo for the third time:
![Screenshot_12](https://user-images.githubusercontent.com/28690790/103410355-2c322600-4b6b-11eb-84a7-ab9b5b11ef75.png)

**Comment by SBPrime** [comment]
I identified and fixed the problem, I will push the fix in next version (3.8.2).

**Comment by x1p** [comment]
Edit, oh wait, I'm dumb, I forgot to replace the default file again :) That's what you get when you perform numerous tests and you forget which config is in place.
Works indeed fine now. Awesome 👍
